docs/source/fundamentals/installation.rst

```diff
@@ -6,11 +6,11 @@ Minimal dependencies
 
 Follow these instructions to install PyTorch, nvFuser, and finally Thunder.
 
-Install PyTorch and nvFuser with pip (command shown is for CUDA 12.1)::
+Install PyTorch and nvFuser with pip (command shown is for CUDA 12.1 and PyTorch 2.4.x)::
 
-  pip install --pre "nvfuser-cu121[torch]" --extra-index-url https://pypi.nvidia.com
+  pip install --pre nvfuser-cu121-torch24
 
-cu121 can be replaced with cu118 depending on your CUDA version.
+cu121 can be replaced with cu118 depending on your CUDA version. NVFuser builds typically support the latest point release of PyTorch stable versions.
```

Comment:
I think we can point to `https://pypi.nvidia.com` as it has wheels for versions other than 11.8 and 12.1.

Reply:
Also, if I follow the link, I see versions with torch and without torch. What is the difference? Which one should I choose?

Reply:
torch 2.3 has cuda 11.8 and 12.1
torch 2.4 has cuda 11.8, 12.1, and 12.4

nvfuser nightly on https://pypi.nvidia.com has the cuda matrix (currently the same as torch 2.4)

Reply:
```suggestion
cu121 can be replaced with cu118 depending on your CUDA version. NVFuser builds typically support the latest point release of PyTorch stable versions.
For torch 2.4, cu124 is also supported. For nightly versions and more detailed instructions, please see https://github.com/NVIDIA/Fuser/#installation
```